learn/advanced/updating.md

```diff
@@ -225,35 +176,192 @@ Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move
+
+```sh
+# replace {dump_uid.dump} with the name of your dump file
+meilisearch --db-path /var/lib/meilisearch/data.ms --import-dump "/var/opt/meilisearch/dumps/{dump_uid.dump}"
 ```
 
+:::
+
+::::
+
 ::: warning
 If you are using Meilisearch v0.20 or below, migration should be done in two steps. First, import your dump into an instance running any version of Meilisearch from v0.21 to v0.24, inclusive. Second, export another dump from this instance and import it to a final instance running your targeted version.
+
+Once Meilisearch v1 is released, this two-step process won't be necessary as v1 will be compatible with dumps from all previous versions.
 :::
 
 Importing a dump requires indexing all the documents it contains. Depending on the size of your dataset, this process can take a long time and cause a spike in memory usage.
 
-Finally, don’t forget to set `displayedAttributes` back to its previous value if necessary. You can do this using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+::: danger `_geo` field in v0.27, v0.28, and v0.29
+You might not be able to import your dump due to Meilisearch allowing malformed `_geo` fields in the above-mentioned versions. Please ensure the `_geo` field follows the [correct format](/learn/advanced/geosearch.md#preparing-documents-for-location-based-search).
+:::
+
+## Step 8: Restart Meilisearch as a service
+
+Once your dump has been correctly imported, press `Ctrl`+`C`  to stop Meilisearch. Next, execute the following command to run the script to configure Meilisearch and restart it as a service:
+
+```
+meilisearch-setup
+```
+
+If required, set `displayedAttributes` back to its previous value using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+
+## Step 9: Delete backup files or rollback
 
-Congratulations! You have successfully migrated your Meilisearch database to the latest version! 🎉
+Now that your Meilisearch instance is up and running, you can delete the backup files with the following commands:
+
+```
+rm /tmp/meilisearch
+rm -r /tmp/data.ms
+```
+
+You can also delete the dump file using:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+rm /path/to/your/meilisearch/directory/meilisearch/dumps/{dump_uid.dump}
+```
+
+:::
+
+::: tab Cloud platforms
+
+```
+rm /var/opt/meilisearch/dumps/{dump_uid.dump}
+```
+
+:::
+
+::::
+
+If for some reason, something went wrong, you can always roll back to the previous version. Move the files back to their previous location using:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /tmp/meilisearch /path/to/your/meilisearch/directory/meilisearch
+mv /tmp/data.ms /path/to/your/meilisearch/directory/meilisearch/data.ms
+```
+
+:::
+
+::: tab Cloud platforms
+
+```
+mv /tmp/meilisearch /usr/bin/meilisearch
+mv /tmp/data.ms /var/lib/meilisearch/data.ms
+```
+
+:::
+
+::::
+
+And run the configuration script:
+
+```
+meilisearch-setup
+```
```

Comment:
Again, pretty sure this only works for cloud platforms, so should go in that tab.

Also, not positive about this, but you may need to specify which directory the user needs to be in to run this command.